sycl/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ if( NOT OpenCL_INCLUDE_DIRS )
   endif()
   ExternalProject_Add(ocl-headers
     GIT_REPOSITORY    https://github.com/KhronosGroup/OpenCL-Headers.git
-    GIT_TAG           origin/master
+    GIT_TAG           d1b936b72b9610626ecab8a991cec18348fba047
```

Comment:
https://github.com/intel/llvm/blob/sycl/opencl-aot/CMakeLists.txt#L20 has similar problem

Reply:
@vladimirlaz thanks! I comment this just after your change in this file.